controllers/producer.go

```diff
@@ -130,10 +134,20 @@ func newProducerConfig(env bootstrap.Environment) (*metric.ProducerConfig, error
 		// init log source port
 		port := env.Config.Global.Misc["logSourcePort"]
 
+		// init initCache
```

Comment:
if we divide the whole improve-plan into several steps, we'd better make it clear and describe the final picture we'll get.

Reply:
improve PR description